src/main/resources/static/vulnerableApp.js

```diff
@@ -105,8 +111,17 @@ function createColumn(detailedInformationArray, key) {
   span.classList.add(
     isSecure ? "secure-variant-tooltip-text" : "unsecure-variant-tooltip-text"
   );
-  span.innerHTML = isSecure ? variantTooltip.secure : variantTooltip.unsecure;
```

Comment:
CWE-79: Improper Neutralization of Input During Web Page Generation ('Cross-site Scripting')
------
User controlled data in methods like `innerHTML`, `outerHTML` or `document.write` is an anti-pattern that can lead to XSS vulnerabilities